src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -203,6 +201,15 @@ private void EnsureWatchingForExit()
             }
         }
 
+        private ProcessWaitHandle GetOrCreateProcessWaitHandle()
+        {
+            if (_waitHandle == null)
+            {
+                _waitHandle = new ProcessWaitHandle(GetWaitState());
+            }
+            return (ProcessWaitHandle)_waitHandle;
+        }
```

Comment:
Nit:
something instead like:
```C#
private ProcessWaitHandle GetOrCreateProcessWaitHandle() =>
    (ProcessWaitHandle)_waitHandle ??= new ProcessWaitHandle(GetWaitState());
```
?

Reply:
:+1: , that will be the first time I use `??=`.